src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -74,36 +67,27 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
       });
     }
 
-    load({
-      deps: [
+    load([
         {{#each sharedJsDepFilenames}}
           '{{../regularBundlePath}}/kbn-ui-shared-deps/{{this}}',
         {{/each}}
-      ],
-      urls: [
-        {
-          deps: [
-            '{{regularBundlePath}}/kbn-ui-shared-deps/{{sharedJsFilename}}',
-            {
-              deps: [
-                '{{dllBundlePath}}/vendors_runtime.bundle.dll.js'
-              ],
-              urls: [
-                {{#each dllJsChunks}}
-                  '{{this}}',
-                {{/each}}
-              ]
-            },
-            '{{regularBundlePath}}/commons.bundle.js',
-          ],
-          urls: [
-            '{{regularBundlePath}}/{{appId}}.bundle.js',
-            {{#each styleSheetPaths}}
-              '{{this}}',
-            {{/each}}
-          ]
-        }
-      ]
+        '{{regularBundlePath}}/kbn-ui-shared-deps/{{sharedJsFilename}}',
+        '{{dllBundlePath}}/vendors_runtime.bundle.dll.js',
+        {{#each dllJsChunks}}
+          '{{this}}',
+        {{/each}}
+        '{{regularBundlePath}}/commons.bundle.js',
+        {{!-- '{{regularBundlePath}}/plugin/data/data.plugin.js', --}}
```

Comment:
Why did you decide to exclude it? 
`watcher` plugin on the current branch is 2.4Mb, on yesterday's version with `data` plugin included in the common bundle it was ~150Kb
![2020-04-08_09-52-09](https://user-images.githubusercontent.com/3198181/78758635-b81d7580-797e-11ea-8fc9-db2ea250fc08.png)

Reply:
Because it has weird side effects that caused bugs and I spent a bunch of time trying to debug them with @lizozom but we couldn't figure out what was causing the `useKibana()` hook to stop working in plugins like graph. We can always follow up and share the data plugin if we figure out how to make it work.